pkg/provider/builder/garden.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package builder
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/gardener/diki/pkg/config"
+	"github.com/gardener/diki/pkg/provider"
+	"github.com/gardener/diki/pkg/provider/garden"
+	"github.com/gardener/diki/pkg/provider/garden/ruleset/securityhardenedshootcluster"
```

Comment:
Let's rename this everywhere. The current directory name seems too long
```suggestion
	"github.com/gardener/diki/pkg/provider/garden/ruleset/securityhardenedshoot"
```